Unix/installbuilder/conf/omiserver.conf

```diff
@@ -33,7 +35,7 @@ httpsport=0
 #NtlmCredsFile=/etc/opt/omi/.creds/ntlm
```

Comment:
Is this the default? Or is this disabled if commented out? Comments indicate that you must comment out the location of the credentials file, just wondering if this is actually the case (in real life) or not. Check with @yakman2020 if unsure.

Reply:
in fact, it is disabled ntlm authentication if commented out.